modin/pandas/test/test_io.py

```diff
@@ -93,9 +99,14 @@ def make_parquet_file():
     Yields:
         Function that generates a parquet file/dir
     """
+    filenames = []
 
     def _make_parquet_file(
-        row_size=NROWS, force=False, directory=False, partitioned_columns=[]
+        filename=TEST_PARQUET_FILENAME,
```

Comment:
update docstring for `filename` argument

Reply:
Docstring was updated.